### PR TITLE
x-kiosk: Update Application

### PR DESCRIPTION
**Pull request by munoz0raul**
- Changed based image to debian:bullseye-slim
- Updated the USER id to weston (63).

### CODE REVIEW

File: x-kiosk/Dockerfile [hunk 1]
```diff
@@ -1,9 +1,36 @@
...
+#
 
-RUN install_packages \
-  sudo \
-  xorg \
-  chromium \
-  lightdm
+# For testing in the LmP
+# docker build --tag hub.foundries.io/lmp/x-kiosk:latest .
+# docker run -it --rm -v /tmp/.X11-unix:/tmp/.X11-unix -v /var/run/dbus:/var/dbus hub.foundries.io/lmp/x-kiosk:latest www.foundries.io
```
Comment (mike-scott) [inline x-kiosk/Dockerfile:18]: Let's fix this line to:
```
# docker run -it --rm -v /tmp/.X11-unix:/tmp/.X11-unix --user "63" hub.foundries.io/lmp/x-kiosk:latest www.foundries.io
```

**Review by mike-scott — CHANGES_REQUESTED** [review]
Looks good except for the testing commands.  Can you verify my change still works?

**Comment by mike-scott** [comment]
And we should squash the 2 commits into 1.

File: x-kiosk/Dockerfile (hunk 1 above)
Comment (munoz0raul) [reply x-kiosk/Dockerfile]: done!

**Comment by munoz0raul** [comment]
@mike-scott, I updated the command to what I have used and squashed the commits. Builds are falling, didn't see if was working before. Do you know if these actions are up-to-date?

**Review by mike-scott — APPROVED** [review]
LGTM

**Comment by mike-scott** [comment]
ci test

**Pull request merged**